fix(protocols): wrong xbil/DEM texture's min and max

gchoqueux opened the pull request:
Wrong xbil/DEM texture's min and max was computed during fetching.

xbil/DEM texture's min and max was only calculated on an entire texture, during fetching.
it didn't consider the smaller pitchs

peppsac commented on src/Provider/WMTSProvider.js:
```diff
@@ -79,11 +80,26 @@ function getXbilTexture(tile, layer, targetZoom) {
+
+        // if pitch is full texture, the min and max are already known
+        if (pitch.z == 1.0 && pitch.w == 1.0) {
+            min = !texture.min ? 0 : texture.min;
```
Is this test `!texture.min` to handle the:
```js
    if (max === -1000000 || min === 1000000) {
        return { min: null, max: null };
    }
```
case from XbilParser?

If yes, then would it make more sense to adapt XbilParser to return `{ max: 0, min: 0 }` in this case instead of fixing here?

gchoqueux replied:
yes, it's changed

gchoqueux committed:
fix(protocols): wrong xbil/DEM texture's min and max

xbil/DEM texture's min and max was only calculated on an entire texture, during fetching.
it did not consider the smaller pitchs

peppsac reviewed (approved):
LGTM, thanks.
